crates/swc_ecma_minifier/src/pass/mangle_names/private_name.rs

```diff
@@ -18,6 +22,8 @@ struct PrivateNameMangler {
     private_n: usize,
 
     renamed_private: AHashMap<Id, JsWord>,
+
+    reserved: Vec<String>,
```

Comment:
Same here.

Reply:
resolved in c12701c9208b636d660ef23458d9aaf87706d2bc